### ISSUE TITLE
Add a configuration property for the active colorHandle / colorHandleRadius

### ISSUE THREAD

**Issue by mheirendt** [jaames/iro.js#166]
It would be useful when there are multiple colors to be able to discern which color is active. This could be as simple as having a configuration property to have a larger colorHandleRadius for the active color as opposed to the handleRadius.

**Comment by jaames** [comment]
Yeah, there is currently a `handleRadius` config option for adjusting global handle radius, so I think it would make sense to add an `activeHandleRadius` for the active color handle specifically. 

Should be an easy addition, I'll look into it for the next update :)

BTW, it's a bit of a hack, but for now can achieve this with some CSS:

```css
// outer circle
.IroHandle--isActive circle:first-of-type {
    r: 11;
}
// inner circle
.IroHandle--isActive circle:last-of-type {
    r: 9;
}
```

**Comment by mheirendt** [comment]
Thank you!

Well done on this library, I assumed you were using an HTML canvas under the hood. SVG makes this change trivial.

**Comment by jaames** [comment]
No worries! 

Thanks :) Yeah, I originally went the HTML5 canvas route early on actually, but found a combo of HTML with CSS gradients and SVG is easier to hack around. I suppose I should advertise that aspect a little more.

**Comment by jaames** [comment]
`activeHandleRadius` will be available in the next release (v 5.5.0). Thanks for the suggestion!
